server/service/src/programs/program_enrolment/program_enrolment_updated.rs

```diff
@@ -24,9 +25,9 @@ pub(crate) fn update_program_enrolment_row(
         .naive_utc();
 
     let repo = ProgramEnrolmentRepository::new(con);
-    let program_row = repo.find_one_by_type_and_patient(&document.r#type, patient_id)?;
-    let id = match program_row {
-        Some(program_row) => program_row.id,
+    let program_enrolment_row = repo.find_one_by_type_and_patient(&document.r#type, patient_id)?;
```

Comment:
Should this be searched by program_id and patient_id now ? It's probably the same but i was a bit confused here about the `r#type`

Reply:
Yes that would fit better now, I changed it in a follow-up PR.